packages/flutter/test/cupertino/magnifier_test.dart

```diff
@@ -108,27 +110,29 @@ void main() {
 
         final BuildContext context = tester.firstElement(find.byType(Placeholder));
 
+        final ValueNotifier<MagnifierInfo> magnifierInfo = ValueNotifier<MagnifierInfo>(
+          MagnifierInfo(
+            currentLineBoundaries: reasonableTextField,
+            fieldBounds: reasonableTextField,
+            caretRect: reasonableTextField,
+            // The tap position is far out of the right side of the app.
+            globalGesturePosition:
+            Offset(MediaQuery.sizeOf(context).width + 100, 0),
+          ),
+        );
+        addTearDown(magnifierInfo.dispose);
         await showCupertinoMagnifier(
           context,
           tester,
-          ValueNotifier<MagnifierInfo>(
-            MagnifierInfo(
-              currentLineBoundaries: reasonableTextField,
-              fieldBounds: reasonableTextField,
-              caretRect: reasonableTextField,
-              // The tap position is far out of the right side of the app.
-              globalGesturePosition:
-                  Offset(MediaQuery.sizeOf(context).width + 100, 0),
-            ),
-          ),
+          magnifierInfo,
```

Comment:
For future changes, to simplify code review, better create late variable:

late MagnifierInfo MagnifierInfo;
addTearDown(() => magnifierInfo.dispose());

...
ValueNotifier<MagnifierInfo>(
  magnifierInfo = MagnifierInfo (...

Reply:
ok